# HH_260123 Internal ESKF diagnostics published for supervisor.
std_msgs/Header header
float32 gnss_innovation_norm
bool gnss_update_accepted
float32 wheel_innovation_norm
bool wheel_update_accepted
float32 covariance_trace
